### ISSUE TITLE
RiveScript produces too much log

### ISSUE THREAD

**Issue by KHaack** [dice-group/sask#69]
the RiveScript tests produces too much log for travis. Travis is limited to a 4MB log if this is exceeded, the build breaks. And RiveScript ignores the log setting :P

```
The log length has exceeded the limit of 4 MB (this usually means that the test suite is raising the same exception over and over).
The job has been terminated
```

In the next branches maven is silent, so it will still work. But in the future the tests should not produce so many logs anymore.

**Comment by prasanthhs** [comment]
Yes i know. But the logs are coming from the Rivescript library. Let me check if there are other options.
